### PR TITLE
add(tools): prettier-plugin-tailwind

### PR DESCRIPTION
**Pull request by Acidic9**
| Name                 | Link                 |
| -------------------- | -------------------- |
| prettier-plugin-tailwind | https://github.com/Acidic9/prettier-plugin-tailwind |

Prettier plugin that sorts a classlist based on tailwind plugins.

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] I have read and followed the [contribution guidelines](.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +176,7 @@
 - 💼 [Tailwind CSS Figma Plugin](https://github.com/impulse/tailwindcss-figma-plugin) - Figma plugin that integrates Tailwind CSS.
 - 💼 [@nuxtjs/tailwindcss](https://github.com/nuxt-community/tailwindcss-module) - Tailwind CSS module for NuxtJS with PurgeCSS and modern CSS (preset env 1).
 - 💼 [preact-cli-tailwind](https://github.com/agneym/preact-cli-tailwind) - Tailwind CSS integration for Preact.
+- 💼 [prettier-plugin-tailwind](https://github.com/Acidic9/prettier-plugin-tailwind) - Prettier plugin that sorts a classlist based on tailwind plugins.
```
Comment (innocenzi) [inline README.md:179]:
```suggestion
- 💼 [prettier-plugin-tailwind](https://github.com/Acidic9/prettier-plugin-tailwind) - Prettier plugin that sorts class lists.
```

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
Really nice, thank you!

**Comment by joshmanders** [comment]
Nice plugin! Didn't know prettier supported plugins now

**Comment by Acidic9** [comment]
@joshmanders thanks! Actually plugins are meant to add support for entirely new languages, but I had to override their html/javascript/typescript language parsers to implement this... hopefully they'll add a proper way to extend the default languages.

**Pull request merged**